app/src/main/java/com/infomaniak/drive/ui/fileList/fileShare/FileShareAddUserDialog.kt

```diff
@@ -45,31 +46,32 @@ import com.infomaniak.lib.core.utils.CoilUtils.simpleImageLoader
 import com.infomaniak.lib.core.utils.Utils.getDefaultAcceptedLanguage
 import com.infomaniak.lib.core.utils.UtilsUi.generateInitialsAvatarDrawable
 import com.infomaniak.lib.core.utils.UtilsUi.getBackgroundColorBasedOnId
-import kotlinx.android.synthetic.main.fragment_bottom_sheet_file_share.*
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
 import java.io.Serializable
 
 class FileShareAddUserDialog : FullScreenBottomSheetDialog() {
 
+    private var binding: FragmentBottomSheetFileShareBinding by safeBinding()
+
     private lateinit var availableUsersAdapter: AvailableShareableItemsAdapter
     private val fileShareViewModel: FileShareViewModel by navGraphViewModels(R.id.fileShareDetailsFragment)
     private val navigationArgs: FileShareAddUserDialogArgs by navArgs()
     private val selectedItems: Share = Share()
 
     private var selectedPermission: Shareable.ShareablePermission = Shareable.ShareablePermission.READ
```

Comment:
Import `Shareable.`

Reply:
Let's do this kind of improvement in another PR where regroup everything